sklearn/datasets/descr/twenty_newsgroups.rst

```diff
@@ -132,13 +141,11 @@ which is fast to train and achieves a decent F-score::
   >>> vectors_test = vectorizer.transform(newsgroups_test.data)
   >>> clf = MultinomialNB(alpha=.01)
   >>> clf.fit(vectors, newsgroups_train.target)
-  MultinomialNB(alpha=0.01, class_prior=None, fit_prior=True)
-
   >>> pred = clf.predict(vectors_test)
   >>> metrics.f1_score(newsgroups_test.target, pred, average='macro')
-  0.88213...
+  0.88213592402729568
```

Comment:
I think that the ellipsis was a good idea. We were just lacking the "doctest: +ELLIPSIS" pragma.